lib/evm_database_ops.rb

```diff
@@ -29,13 +29,13 @@ def self.database_size(opts)
     PostgresAdmin.database_size(opts)
   end
 
-  def self.validate_free_space(database_opts)
-    free_space = backup_destination_free_space(database_opts[:local_file])
+  def self.validate_free_space(target_uri, dbname)
```

Comment:
Mentioned this in the commit message, but I change this just to make this a little easier to digest.  The only usages of this method (in this repo at least) is what was found below.

I can be (quite easily) convinced to not make the change to this method signature to make this a more targeted fix for now, but wanted to provide it with the fix on the first pass.  Can be easily reverted if the reviewer(s) think it is too risky for a backport.